cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/LedgerEvents/Gov/DRepActivity.hs

```diff
@@ -191,46 +188,11 @@ activityChangeProposalTest execConfig epochStateView configurationFile socketPat
   (EpochNo epochAfterProp) <- getCurrentEpochNo epochStateView
   H.note_ $ "Epoch after \"" <> prefix <> "\" prop: " <> show epochAfterProp
 
-  waitAndCheck epochAfterProp
+  waitAndCheckNewEpochState epochStateView configurationFile socketPath ceo minWait (EpochInterval <$> mExpected) maxWait
```

Comment:
For instance why do we care about the minimum wait here? Surely we are only interested that the value we are looking for appears in the ledger state and that the function terminates at a some point in the future.

Reply:
See my other comment: https://github.com/IntersectMBO/cardano-node/pull/5815#discussion_r1589515167